AppCenterAnalytics/AppCenterAnalytics/Internals/Session/MSACSessionTracker.h

```diff
@@ -46,6 +46,16 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)stop;
 
+/**
+ * Manual session tracking checking.
```

Comment:
`Manual session tracking checking.` -> `Start a new session if manual session tracker is enabled, otherwise do nothing.`